IdentityCore/tests/integration/ios/MSIDDefaultSilentTokenRequestTests.m

```diff
@@ -1594,23 +1514,15 @@ - (void)testAcquireTokenSilent_whenFamilyRefreshTokenInCache_andClientMismatch_s
     
     [silentRequest executeRequestWithCompletion:^(MSIDTokenResult * _Nullable result, NSError * _Nullable error) {
         
-        XCTAssertNil(error);
-        XCTAssertNotNil(result);
-        XCTAssertEqualObjects(result.accessToken.accessToken, @"new at mrrt");
-        XCTAssertEqualObjects(result.accessToken.scopes, [NSOrderedSet msidOrderedSetFromString:@"user.read tasks.read"]);
-        XCTAssertEqualObjects(result.account.accountIdentifier.homeAccountId, silentParameters.accountIdentifier.homeAccountId);
-        XCTAssertEqualObjects(result.rawIdToken, [MSIDTestIdTokenUtil idTokenWithPreferredUsername:DEFAULT_TEST_ID_TOKEN_USERNAME subject:@"sub" givenName:@"Test" familyName:@"User" name:@"Test Name" version:@"2.0" tid:DEFAULT_TEST_UTID]);
-        XCTAssertFalse(result.extendedLifeTimeToken);
-        NSURL *tenantURL = [NSURL URLWithString:@"https://login.windows.net/"DEFAULT_TEST_UTID];
-        XCTAssertEqualObjects(result.authority.url, tenantURL);
-        XCTAssertEqualObjects(result.refreshToken.refreshToken, @"new mrrt");
+        XCTAssertNil(result);
+        XCTAssertNotNil(error);
```

Comment:
could you provide a bit more information why this test case was changed?

Reply:
For now, I will just make the test pass, but will take a closed look once the main change looks good. 

To answer your question regarding this change, the test was renamed to ```testAcquireTokenSilent_whenFamilyRefreshTokenInCache_andNoAppRefresh_ClientMismatch_shouldSkip```

If we keep the original test, it will first try with ART and succeed in new flow, and will not able to shows clientMismatch_shouldSkip. So I removed the ART and make it tests against FRT directly which generates the error and no results

Reply:
UT has been updated